addon/utils/url-builder.js

```diff
@@ -1,8 +1,17 @@
-import param from 'jquery-param';
+import URL from 'url';
+import { isEmpty } from '@ember/utils';
 
 export default function(url, path, queryParams) {
-  let query = param(queryParams);
-  let pathUrl = url.charAt(url.length - 1) === '/' ? `${url}${path}` : `${url}/${path}`;
-
-  return query ? `${pathUrl}?${query}` : pathUrl;
+  let urlObj = new URL(url, 'https://example.com');
```

Comment:
What if the url is absolute? Previously this function always returned absolute URL, while now its relative.

Reply:
Perhaps I'm the one who led us down the wrong path here, in my [previous comment](https://github.com/Exelord/ember-custom-actions/pull/53#discussion_r412908658).

If `url` is always absolute something like `let urlObj = new URL(url);` should work, along with `return urlObj.toString();`

Reply:
I'm editing the PR in order to use native URL API only for queryParams and keep something more raw for the pathUrl to avoid conflict with absolute/relative URLs
Thanks for your feedbacks